slink-main/index.js

```diff
@@ -13,13 +14,18 @@ module.exports.handler = async (event, context, callback) => {
     // Load all configuration parameters from AWS SSM
     await config.loadConfigParams(context);
 
-    const result = await introduction.process();
+    const introductionResult = await introduction.process();
+    const activationResult = await activation.process();
 
     await writeRunRecord(context);
 
     const response = {
       statusCode: 200,
-      body: JSON.stringify({ message: `Sent ${result.successful} candidate(s) to SAP, failed: ${result.unsuccessful}` })
+      body: JSON.stringify({
+        message: `Sent ${introductionResult.successful} candidate(s) to SAP, failed: ${introductionResult.unsuccessful}. ` +
+                 `Activated ${activationResult.successful} candidate(s) in SAP, ` +
+                   `failed to activate: ${activationResult.unsuccessful}`
+      })
```

Comment:
You may not have had time yet, but I _think_ we should combine the results of introduction and activation into an object something like this:

```
{
  statusCode: 200,
  body: {
    results/details/whatever: {
      introduction: { /* introduction function return object */ }
      activation: { /* activation function return object */ }
    }
    message: 'some useful statistics'
  }
}
```

And I don't know _WHY_ I was JSON.stringifying the results, but I don't think we want/need to do that.  Must have been in a fever dream.  My bad :-).